app/src/main/java/com/example/playlistmaker/ui/settings/SettingsViewModel.kt

```diff
@@ -38,19 +38,19 @@ class SettingsViewModel(
         darkTheme.value?.let { ThemeSwitcher.switchTheme(it) }
     }
 
-    fun callSendIntent(message: String){
-        externalNavigator?.intentSend(message)
+    fun callSendIntent(message: String): String?{
```

Comment:
🍏 В Kotlin можно применять следующий синтаксис:
```suggestion
    fun callSendIntent(message: String): String? = externalNavigator?.intentSend(message)
```